daemon/daemon.go

```diff
@@ -63,19 +65,37 @@ func (ib *Infoblox) Allocate(args *ExtCmdArgs, result *types.Result) (err error)
 
 	log.Printf("RequestAddress: '%s', '%s', '%s'", netviewName, subnet, mac)
 	ip, _ := ib.Drv.RequestAddress(netviewName, subnet, "", mac, args.ContainerID)
-
+        log.Printf("Allocated IP: '%s'", ip)
+        hwAddr, err := hwaddr.GenerateHardwareAddr4(net.ParseIP(ip), hwaddr.PrivateMACPrefix)
```

Comment:
Sorry, I'm not familiar with the CNI 0.3.1 spec....

Question: is the MAC not provided in the request anymore? Why are we using a generated MAC all the time?